src/features/unified-share-modal/ContactsField.js

```diff
@@ -28,15 +28,15 @@ type Props = {
     onContactAdd: Function,
     onContactRemove: Function,
     onInput?: Function,
-    onSuggestedCollaboratorAdd: Contact => void,
     selectedContacts: Array<Contact>,
-    suggestedCollaborators?: Array<Contact>,
+    suggestedCollaborators?: Object,
```

Comment:
Is it possible to define type for this?

Reply:
I can make it more specific. I'll have to check the flow docs. It basically an Object where every key and value is the same format.

Reply:
Update with improved flow typing